lib/template.hb.swift

```diff
@@ -53,4 +53,40 @@ extension UIKitStyleContainer {
         return container
     }
 }
+
+/// Casing support to enable casing in iOS 14 only.
+extension Text {
```

Comment:
Do we need this, aka what’s wrong with Text.Case ?

Reply:
Yes, because we generate the same code for iOS 13/14, if we used `Text.Case` the generated stylist implementation wouldn't compile on iOS 13 (or any project that supported iOS 13) with `'Case' is only available in iOS 14.0 or newer`. This way, the stylist compiles and in iOS 13 simply ignores the casing.